javascripts/content_scripts/posting.floating.js

```diff
@@ -183,7 +183,6 @@ if (SeamlessPosting === undefined) {
     var node = this.getNode();
     node.style.transition = 'none'; // remove transition
     this.updateVisibility();  // reset animation to the hidden state
-    node.offsetWidth;   // force reflow
```

Comment:
This is a semantic change that would require extensive testing. It is probably best to not include it here.